api/v1alpha1/etcdpeer_types_test.go

```diff
@@ -0,0 +1,49 @@
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidateCreate(t *testing.T) {
+	t.Run("success", func(t *testing.T) {
```

Comment:
can you follow the naming conventions in https://github.com/improbable-eng/etcd-cluster-operator/blob/master/docs/contributing.md ?

i.e. CamelCase separated with `/` or `_`